packages/mobile/locales/es-419/inviteFlow11.json

```diff
@@ -22,5 +22,11 @@
     "title": "Invita a un amigo",
     "body": "Conecta y comparte valor con tus amigos y familia en {{appName}}.",
     "button": "Invitar"
+  },
+  "sendAndInvite": "Envía e invita",
+  "inviteAndSendDialog": {
+    "title": "Invita y envía",
+    "body": "Tu pago a {{name}} estará esperando por ellos hasta que verifiquen su número de teléfono. ¡Envíales un mensaje de invitación para que se registren en {{appName}} y canjeen tu pago!",
+    "button": "Invitar"
```

Comment:
@gnardini could you confirm this? 🙏

Reply:
It's mostly OK, a small thing I'd change is in the body text:
`¡Envíales un mensaje de invitación para que se registren en {{appName}} y canjeen tu pago!`
->
`¡Invítalos a registrarse en {{appName}} para que puedan recibir tu pago!`

Reply:
¡Envíales una invitación para que se registren en {{appName}} y reciban tu pago!

Reply:
> Tu pago a {{name}} estará esperando por ellos hasta que verifiquen su número de teléfono. 
->
> Tu pago estará esperando hasta que {{name}} verifique su número de teléfono.